### ISSUE TITLE
Save to Web Service failure

### ISSUE THREAD

**Issue by mryanlord** [MDMI/mdmirieditor#4]
got an error from the RI Editor when trying to save an RI revision to the web

org.openhealthtools.mdht.mdmi.MdmiException: Proxy set() failed!

org.openhealthtools.mdht.mdmi.service.MdmiBusinessElementProxy.set(MdmiBusinessElementProxy.java:294)
org.openhealthtools.mdht.mdmi.service.MdmiBusinessElementProxy.set(MdmiBusinessElementProxy.java:270)
org.openhealthtools.mdht.mdmi.editor.io.WebServiceUtillity.saveAllMdmiBusinessElementReferences(WebServiceUtillity.java:111)
org.openhealthtools.mdht.mdmi.editor.actions.SaveChangesAction.updateBERs(SaveChangesAction.java:269)
org.openhealthtools.mdht.mdmi.editor.actions.SaveChangesAction.saveChanges(SaveChangesAction.java:76)
org.openhealthtools.mdht.mdmi.editor.actions.SaveChangesAction.execute(SaveChangesAction.java:49)
org.openhealthtools.mdht.mdmi.editor.actions.AbstractAction.actionPerformed(AbstractAction.java:83)
javax.swing.AbstractButton.fireActionPerformed(Unknown Source)
javax.swing.AbstractButton$Handler.actionPerformed(Unknown Source)
javax.swing.DefaultButtonModel.fireActionPerformed(Unknown Source)
javax.swing.DefaultButtonModel.setPressed(Unknown Source)
javax.swing.AbstractButton.doClick(Unknown Source)
javax.swing.plaf.basic.BasicMenuItemUI.doClick(Unknown Source)
javax.swing.plaf.basic.BasicMenuItemUI$Handler.mouseReleased(Unknown Source)
java.awt.Component.processMouseEvent(Unknown Source)
javax.swing.JComponent.processMouseEvent(Unknown Source)
java.awt.Component.processEvent(Unknown Source)
java.awt.Container.processEvent(Unknown Source)
java.awt.Component.dispatchEventImpl(Unknown Source)
java.awt.Container.dispatchEventImpl(Unknown Source)
java.awt.Component.dispatchEvent(Unknown Source)
java.awt.LightweightDispatcher.retargetMouseEvent(Unknown Source)
java.awt.LightweightDispatcher.processMouseEvent(Unknown Source)
java.awt.LightweightDispatcher.dispatchEvent(Unknown Source)
java.awt.Container.dispatchEventImpl(Unknown Source)
java.awt.Window.dispatchEventImpl(Unknown Source)
java.awt.Component.dispatchEvent(Unknown Source)
java.awt.EventQueue.dispatchEventImpl(Unknown Source)
java.awt.EventQueue.access$500(Unknown Source)
java.awt.EventQueue$3.run(Unknown Source)
java.awt.EventQueue$3.run(Unknown Source)
java.security.AccessController.doPrivileged(Native Method)
java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
java.awt.EventQueue$4.run(Unknown Source)
java.awt.EventQueue$4.run(Unknown Source)
java.security.AccessController.doPrivileged(Native Method)
java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(Unknown Source)
java.awt.EventQueue.dispatchEvent(Unknown Source)
java.awt.EventDispatchThread.pumpOneEventForFilters(Unknown Source)
java.awt.EventDispatchThread.pumpEventsForFilter(Unknown Source)
java.awt.EventDispatchThread.pumpEventsForHierarchy(Unknown Source)
java.awt.EventDispatchThread.pumpEvents(Unknown Source)
java.awt.EventDispatchThread.pumpEvents(Unknown Source)
java.awt.EventDispatchThread.run(Unknown Source)

**Comment by mryanlord** [comment]
I tried to save the RI after making the total changes, and I got this error. 

I tried with just one change to one eleement - changed the name of AllergySectionText to AllergySectionNotes. No problems. 

I double checked the revise csv to make sure all the data in the unique id fields was actually unique IDs (this had been done to compare the revised version, which was exported into a CSV fine), and they were. 

I tried rebooting the RI editor to see if the changes kept, and the error message was an error itself; the changes were not present.
